AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/allocate-hearings/allocate-hearings.component.ts

```diff
@@ -206,19 +244,23 @@ export class AllocateHearingsComponent implements OnInit {
     private updateTableWithAllocatedCso(updatedHearings: AllocationHearingsResponse[]) {
         this.allocationHearingViewModel.updateHearings(updatedHearings);
         this.allocationHearingViewModel.uncheckAllHearingsAndRevert();
-        this.csoAllocatedMenu.clear();
+        this.selectAllocateCso.clear();
         this.updateMessageAndDisplay(Constants.AllocateHearings.ConfirmationMessage);
     }
 
     selectHearing(checked: boolean, hearing_id: string) {
         if (checked) {
             this.clearHearingUpdatedMessage();
-            const csoUsername = this.csoAllocatedMenu?.selectedLabel;
-            // safe to cast to string
-            const csoId = this.csoAllocatedMenu?.selectedItems as string;
-
             this.allocationHearingViewModel.checkHearing(hearing_id);
-            this.attemptToAssignCsoToSelectedHearings(csoId, csoUsername);
+
+            const selectedCso = this.selectAllocateCso?.selected as SelectOption;
+            if (selectedCso) {
+                console.log('selectedCso', selectedCso);
```

Comment:
can probably remove this, or alternatively use `Logger` instead

Reply:
Done